### PR TITLE
Made a few changes on CSS and Plans page

### PR DESCRIPTION
**Pull request by Jenna-lawrence**
Made a few changes on CSS and Plans page

### CODE REVIEW

File: src/main/php/plans.php [hunk 1]
```diff
@@ -20,54 +20,44 @@
...
-<?php include("universal-header.php");  ?>
-    <div class="plans-header">
-        <h1>Learn about our plans here!</h1>
-        <br
+    <?php include("universal-header.php");  ?>
+
+    <h1>Learn about our plans here!</h1>
+
+    <div class="plans-block">
+        <img src="https://images.theconversation.com/files/339674/original/file-20200604-130929-iutdtd.jpg?ixlib=rb-1.1.0&rect=8%2C0%2C5455%2C3637&q=45&auto=format&w=496&fit=clip" alt="Bronze">
```
Comment (rgb20206) [inline src/main/php/plans.php:28]: can we make this an internal resource?

File: src/main/php/plans.php [hunk 2]
```diff
@@ -20,54 +20,44 @@
...
+        <ul>
+            <li>Our silver plan is the best for value!</li>
+            <li>Get free access to the swimming pool</li>
+            <li>1 free day pass per month for a friend</li>
+            <li>Free personal trainer sessions twice a month</li>
+        </ul>
+    </div>
+
+    <div class="plans-block">
+        <img src="https://specials-images.forbesimg.com/imageserve/5f298efbe2252631e7028543/960x0.jpg?fit=scale" alt="gold">
```
Comment (rgb20206) [inline src/main/php/plans.php:48]: and here.

File: src/main/resources/css/style.css [hunk 3]
```diff
@@ -258,37 +258,48 @@ input {
     min-height: 100vh;
     position: relative;
     width: 100%;
-    background-image: url(../media/background-3.png);
+    background-image: url(https://static01.nyt.com/images/2020/01/06/well/04run-gym/04run-gym-superJumbo.jpg);
```
Comment (rgb20206) [inline src/main/resources/css/style.css:261]: and here.

File: src/main/php/plans.php [hunk 4]
```diff
@@ -20,54 +20,44 @@
...
-                    <ul>
-                        <li>Our bronze plan is the cheapest!</li>
-                        <li>Unlimited access to the gym</li>
-                        <br>
-                    </ul>
-                </div>
-            </div>
-            <div class="Width">
-                <img src="https://static01.nyt.com/images/2020/01/06/well/04run-gym/04run-gym-superJumbo.jpg" alt="Silver" class="plans-image-2" >
-            <p>Sliver</p>
```
Comment (Arran-Mc) [inline src/main/php/plans.php:43]: Small typo!

**Review by rgb20206 — APPROVED** [review]
looks good.

File: src/main/php/plans.php (hunk 4 above)
Comment (rgb20206) [reply src/main/php/plans.php:43]: this code was removed hence the red :).
additions are below.

Comment (Jenna-lawrence) [reply src/main/php/plans.php:43]: this code was removed hence the red :).
additions are below.

**Pull request merged**